### DISCUSSION TITLE
openquestions/eliminating-hyperparameters/

### DISCUSSION THREAD

**Discussion by giscus** [learningmechanics/learningmechanics.github.io#10] [Comments]
# openquestions/eliminating-hyperparameters/

Discussion page for open question: Can we understand and eliminate all hyperparameters?

https://learningmechanics.pub/openquestions/eliminating-hyperparameters/

**Comment by AMindToThink** [comment]
Does eliminating hyperparameters make sense? I'm under the impression that there's always going to be a tradeoff between model size and loss, and overfitting vs underfitting.
Does 'eliminating hyperparameters' mean something more like 'make a function from desired model properties to hyperparameters'?
